src/controllers/addserver.js

```diff
@@ -46,6 +46,16 @@ define(["appSettings", "loading", "browser", "emby-button"], function(appSetting
         view.querySelector(".addServerForm").addEventListener("submit", onServerSubmit);
         view.querySelector(".btnCancel").addEventListener("click", goBack);
 
+        // HACK: To catch the end of loading
```

Comment:
eh, what this changes?

Reply:
At dedicated Tizen app project (dmitrylyzo/jellyfin-tizen) I catch 'end of loading' by watching for loading indicator 'hide' style class. Actually it is better to have some event from viewManager to call my onPageLoad.

Reply:
mhm, I see... can you introduce a new event in this case instead of hijacking the loading indicator?..